src/lib/components/custom/theme-selector.svelte

```diff
@@ -78,20 +67,38 @@
 		-webkit-appearance: none;
 		-moz-appearance: none;
 		appearance: none;
+		visibility: hidden;
 		cursor: pointer;
 		border: none;
-		background-color: transparent;
+		background-color: var(--colors-top);
+		padding: 0;
+		width: 100%;
+		height: 100%;
+	}
+	input[type='color']::-webkit-color-swatch-wrapper {
+		box-sizing: border-box;
+		margin: 0;
+		border: 0;
 		padding: 0;
-		width: 3rem;
-		height: 3rem;
 	}
 	input[type='color']::-webkit-color-swatch {
+		margin: 0;
+		outline: 0;
 		border: none;
-		border-radius: var(--border-radius);
+		border-radius: 0 var(--border-radius) var(--border-radius) 0;
+		padding: 0;
+		width: 100%;
+		height: 100%;
 	}
 	input[type='color']::-moz-color-swatch {
+		display: flex;
+		margin: 0;
+		outline: 0;
 		border: none;
-		border-radius: var(--border-radius);
+		border-radius: 0 var(--border-radius) var(--border-radius) 0;
+		padding: 0;
+		width: 100%;
+		height: 100%;
 	}
```

Comment:
These look pretty much the same for the webkit and mozilla version. Can it be unified and if needed the distinct css attributes added only the the relevant selector (here AFAIK it is just `display: flex` in mozilla)

```css
input[type='color']::-webkit-color-swatch,
input[type='color']::-moz-color-swatch {
		margin: 0;
		outline: 0;
		border: none;
		border-radius: 0 var(--border-radius) var(--border-radius) 0;
		padding: 0;
		width: 100%;
		height: 100%;
	}
	input[type='color']::-moz-color-swatch {
		display: flex;
	}
	```

Reply:
BTW They can be identical, and also currently the color input is invisible so they are not strictly needed, unless we want to show the selected color in the button.